modules/parquet/package.json

```diff
@@ -42,6 +42,7 @@
     "@loaders.gl/compression": "3.2.0-alpha.1",
     "@loaders.gl/loader-utils": "3.2.0-alpha.1",
     "@loaders.gl/schema": "3.2.0-alpha.1",
+    "apache-arrow": "^7.0.0",
```

Comment:
Let's discuss... arrow 7 is a breaking change. I have a PR that updates to arrow 7 but have been holding it off for loaders.gl 4.0.

Reply:
Or maybe apache-arrow a peer dependency? The only functions used are `arrow.tableFromIPC` (for reading) and `arrow.tableToIPC` (for writing). Arrow 7 isn't necessary; I just did `yarn add apache-arrow` and it chose the latest version

Reply:
The problem is that those functions have changed name compared to pre v7 releases. v7 was a major refactor that massively improves tree-shaking of arrow but there are a bunch of breaking changes.

Reply:
Yes, I dropped the dev dependencies version down to `^4` and had to reimplement both `tableToIPC` and `tableFromIPC` using the lower level `RecordBatchFileWriter` and `RecordBatchStreamReader`.
https://github.com/visgl/loaders.gl/blob/26317f7a066032a1c833810e0224d3c22a988b4c/modules/parquet/src/lib/encode-parquet-wasm.ts#L26-L28
https://github.com/visgl/loaders.gl/blob/26317f7a066032a1c833810e0224d3c22a988b4c/modules/parquet/src/lib/parse-parquet-wasm.ts#L21-L28